broadinstitute/pearl
# [AR-141] Data Dictionary

How to test:

1. Restart AdminApiApp
2. Repopulate ourhealth `./scripts/populate_portal.sh ourhealth`
3. Query local Postgres to see data in new table: 
`docker exec -it postgres psql pearl -U postgres -c 'select * from survey_question_definition;'`

I found that viewing the entire table looked a bit wonky on my screen because of the `choices` JSON column, so here's another one-liner to exclude it:
`docker exec -it postgres psql pearl -U postgres -c 'select id, survey_id, survey_stable_id, survey_version, question_stable_id, question_type, required, created_at, last_updated_at from survey_question_definition;'`
